bedrock/mozorg/helpers/misc.py

```diff
@@ -517,3 +518,8 @@ def slugify(text):
     trailing whitespace.
     """
     return django_slugify(text)
+
+
+@jingo.register.filter
+def bleach_tags(text):
+    return bleach.clean(text, tags=[], strip=True).replace('&amp;', '&')
```

Comment:
Maybe `bleach_tags` should return a safe string so we don't need all these safe filters? Is bleach the thing converting ampersands?